examples/paraToParaTransferMultiAsset.ts

```diff
@@ -0,0 +1,44 @@
+/**
+ * When importing from @substrate/asset-transfer-api it would look like the following
+ *
+ * import { AssetTransferApi, constructApiPromise } from '@substrate/asset-transfer-api'
+ */
+import { AssetTransferApi, constructApiPromise } from '../src';
+import { TxResult } from '../src/types';
+import { GREEN, PURPLE, RESET } from './colors';
+
+/**
+ * In this example we are creating a call to send BNC from a Moonriver (Parachain) account
+ * to a Bifrost Kusama (Parachain) account, where the `xcmVersion` is set to 2, and `isLimited` is false declaring that
+ * it will allow `unlimited` weight for the tx.
+ *
+ * NOTE: When `isLimited` is true it will expect for refTime and proofSize to be provided as additional arguments.
```

Comment:
Same with this as above, if `isLimited` is true and no `weightLimit` is passed in, it will be unlimited.